scripts/versions.sh

```diff
@@ -3,5 +3,5 @@
 # Set up the versions to be used
 subnet_evm_version=${SUBNET_EVM_VERSION:-'v0.2.9'}
 # Don't export them as they're used in the context of other calls
-avalanche_version=${AVALANCHE_VERSION:-'v1.7.16'}
+avalanche_version=${AVALANCHE_VERSION:-'v1.7.18'}
 network_runner_version=${NETWORK_RUNNER_VERSION:-'v1.2.0'}
```

Comment:
seems current ANR version is not compatible with 1.7.18

Reply:
logging lib updates for avago 1.7.17 conflicts with ANR 1.2.0 at E2E compile time (or lint). it is solved in darioush PR by using the latest ANR